sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
We should not rely on the implementation of `_eval_is_real()` (which is currently wrong).